solr_response.go

```diff
@@ -29,12 +29,10 @@ func GetDocIdFromDoc(m map[string]interface{}) string {
 func GetVersionFromDoc(m map[string]interface{}) int {
 	if v, ok := m["_version_"]; ok {
 		switch v := v.(type) {
-		default:
-			return 0
```

Comment:
why remove the default

Reply:
Theres a `return 0` at the end anyways, so if the value is neither an int nor a float64, the final return will be evaluated anyways meaning the `default` is redundant